src/mug/styles/core.clj

```diff
@@ -39,8 +39,9 @@
    [:h1 :h1 :h3 :h4 :h5 :h6 {:font-size   "100%"
                              :font-weigth "normal"}]
    [:* {:font-family "'Open Sans', sans-serif"}]
-
-   [:.app {:height "100%"
+   [:.height-zero {:height "0"}]
+   [:.width-zero  {:width "0"}]
```

Comment:
>.height-zero 
>.width-zero 

Here is multiple issues:
1. Why do we need this styles at all? How many elements will contain such classes to put it in global scope?
2. It's better to use "0px" for consistency.
3. It looks like that `.width-zero` is not used.

And if we really need to have such styles, it's better to define them as "modifiers" where it's necessary: https://jsfiddle.net/f4w2z8s3/